app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,134 @@
+class Api::V1::UsersController < ApiController
+
+  include Swagger::Blocks
+
+  swagger_path '/users' do
+    operation :get do
+      key :summary, 'Get all users for the membership'
+      key :description, 'List all users'
+      key :operationId, 'getUsers'
+      security do
+        key :api_token, []
+      end
+      key :tags, ['Users']
+
+      extend Api::SwaggerResponses
+      include_404_schema
+      include_success_schema(model: 'User', format: :array)
+    end
+  end
+  def index
+    render api_response data: current_member.users
+  end
+
+  swagger_path '/users/add' do
```

Comment:
well that's the thing.. since we're using tokens there's no such thing as a "power user" in regarding the api.  We should discuss further, but right now I'm only allowing power users to access the management web interface.